network/requestTracker.go

```diff
@@ -501,3 +507,13 @@ func (rt *RequestTracker) getForwardedConnectionAddress(header http.Header) (ip
 	}
 	return
 }
+
+// isLocalhost returns true if the given host is a localhost address.
+func isLocalhost(host string) bool {
+	for _, v := range []string{"localhost", "127.0.0.1", "[::1]", "::1"} {
```

Comment:
Go is also currently representing localhost as "[::]"
(this is what I see in a algod.net, `[::]:4160`)